src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(Colors randomColor, int i) {
+        this.color = color;
+        this.number = number;
+    }
```

Comment:
It will not work properly in current state as you accept variable "i" but trying to assign "number" and same for "randomColor" and "color" (but here also you have type mismatch)
```suggestion
    public Ball(String color, int number) {
        this.color = color;
        this.number = number;
    }
```